frontend/javascripts/admin/tasktype/task_type_create_view.tsx

```diff
@@ -43,464 +63,446 @@ function isValidMagnification(_rule, value) {
+                            value,
+                            getFieldValue(["settings", "resolutionRestrictions", "max"]),
+                          ),
+                      },
+                    ]}
                   >
-                    <FormItem
-                      name={["settings", "resolutionRestrictionsForm", "min"]}
-                      hasFeedback
-                      label="Minimum"
-                      style={{
-                        marginBottom: 6,
-                      }}
-                      rules={[
-                        {
-                          validator: isValidMagnification,
-                        },
-                      ]}
-                    >
-                      <InputNumber min={1} size="small" disabled={isEditingMode} />
-                    </FormItem>
-                    <FormItem
-                      name={["settings", "resolutionRestrictionsForm", "max"]}
-                      hasFeedback
-                      label="Maximum"
-                      rules={[
-                        {
-                          validator: isValidMagnification,
-                        },
-                      ]}
-                    >
-                      <InputNumber min={1} size="small" disabled={isEditingMode} />
-                    </FormItem>
-                  </div>
-                ) : null
-              }
-            </FormItem>
-
-            <FormItem>
-              <RecommendedConfigurationView
-                // @ts-expect-error ts-migrate(2322) FIXME: Type 'FormInstance<any> | null' is not assignable ... Remove this comment to see the full error message
-                form={form}
-                enabled={this.state.useRecommendedConfiguration}
-                onChangeEnabled={this.onChangeUseRecommendedConfiguration}
-              />
-            </FormItem>
-
-            <FormItem>
-              <Button type="primary" htmlType="submit">
-                {`${titlePrefix} Task Type`}
-              </Button>
-            </FormItem>
-          </Form>
-        </Card>
-      </div>
-    );
-  }
+                    <InputNumber min={1} size="small" disabled={isEditingMode} />
+                  </FormItem>
+                  <FormItem
+                    name={["settings", "resolutionRestrictions", "max"]}
+                    hasFeedback
+                    label="Maximum"
+                    rules={[
+                      {
+                        validator: isValidMagnification,
+                      },
+                      {
+                        validator: (_rule, value) =>
+                          isMaximumMagnificationSmallerThenMinRule(
+                            value,
+                            getFieldValue(["settings", "resolutionRestrictions", "min"]),
+                          ),
+                      },
+                    ]}
+                  >
+                    <InputNumber min={2} size="small" disabled={isEditingMode} />
```

Comment:
Yes, you are right. That being said, this is exactly what the old code (lines 57ff) checked . If you only have one resolution one would simply not limit the resolutions for a task type at all, right?

https://github.com/scalableminds/webknossos/pull/7702/files#diff-ea5bab681d8de7e27916f517abf5d3ab2cf72dc4f2592c0740eb0cc7d9955133L57

Reply:
cc @philippotto

Reply:
You mean this section here, right? (The link does not work for me)
```javascript
if (
    resolutionRestrictions.min != null &&
    resolutionRestrictions.max != null &&
    resolutionRestrictions.min > resolutionRestrictions.max
  ) {
    Toast.error("Minimum resolution must not be greater than maximum resolution.");
    return null;
  }
  ```
  To me this code reads like `min=1` and `max=1` is allowed but not `min=2` and `max=1` because the if only rejects in case `min > max` and not in case they are equal (which would be `min >= max` or ` resolutionRestrictions.min >= resolutionRestrictions.max`). Therefore, I'd say that the previous code allowed setting `min=1` and `max=1`, but the your code changes with the InputNumber for the maximum field required the maximum to be at least `2>=` and thus  `min=1` and `max=1` wouldn't be possible anymore. (Although I haven't tested the changes yet and only looked at the code)

Reply:
Good catch.